src/api/libcellml/generator.h

```diff
@@ -63,27 +63,40 @@ class LIBCELLML_EXPORT GeneratorVariable
     static GeneratorVariablePtr create() noexcept;
 
     /**
-     * @brief Get the @c Variable for this @c GeneratorVariable.
+     * @brief Get the initial value @c Variable for this @c GeneratorVariable.
```

Comment:
maybe instead of "initial value @c Variable" we could have "initialising @c Variable" ?  I found it hard to follow on my first reading?  Same - perhaps - with the function name?

Reply:
I was trying to stick to the CellML linguo, but I agree that we don't need to (and probably shouldn't in fact!). Ok, changed in commit 534d064a.

Otherwise, what do you mean by the function name?